packages/fxa-settings/src/components/AlertExternal/index.tsx

```diff
@@ -8,6 +8,7 @@ export const AlertExternal = () => {
 
   return account.alertTextExternal ? (
     <AlertBar
+      type="info"
```

Comment:
Shouldn't this actually be success?

Reply:
It could be anything, really, since all messages are able to be sent from content server. I was just going the neutral route.

Reply:
Right now, the only messages we convey are the "verified email success" messages, which should be green. I just hit `beta/settings` without an account, created one, and see this: (also, eep, I'm getting a `Attempted to mount AlertBar on element ID "alert-bar-root" but it was not found in the document` error here which is also why it looks a little cut off, if it can't be fixed easily here let's file a follow up?)
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/13018240/91914855-ac5c8380-ec7e-11ea-84e4-2c309eb95fe4.png">

Reply:
I can dig it, will change to success (actually, I'll just remove it to default). Will look at that other thing in the morning.

Reply:
Ehh, I've been looking at this for a bit, but don't quite understand how it might be happening. AlertExternal is only ever invoked within the initial AlertBarContent provider set, so I'm not sure why the ref isn't being passed down in this specific context. This is filed here: https://jira.mozilla.com/browse/FXA-2505.